frontend/apps/filemanager/filemanagercollection.lua

```diff
@@ -105,10 +105,16 @@ function FileManagerCollection:updateItemTable(show_last_item, item_table)
     local collection_name = self:getCollectionTitle(self.coll_menu.collection_name)
     local title = T("%1 (%2)", collection_name, #item_table)
     local subtitle = ""
-    if self.match_table ~= nil then
+    if self.match_table then
         subtitle = {}
-        for prop, value in pairs(self.match_table.props) do
-            table.insert(subtitle, self.ui.bookinfo.prop_text[prop] .. " " .. value)
+        if self.match_table.status then
+            local status_string = BookList.getBookStatusString(self.match_table.status, true)
+            table.insert(subtitle, "\u{0000}" .. status_string) -- sorted first
```

Comment:
Ha! Neat ;).

Reply:
(Assume the font renders a NUL as a zero-width glyph, or not at all, of course).

Reply:
Looks like it's zero-width.
See screenshots in https://github.com/koreader/koreader/pull/12981.
"N/A" in the list of tags has the same symbol.
https://github.com/koreader/koreader/blob/54add8e495a04b02646a04c32f981c04059dd02d/frontend/apps/filemanager/filemanagercollection.lua#L25